frontend/src/components/TableUnratedTaskRow.tsx

```diff
@@ -194,14 +194,16 @@ export const TableUnratedTaskRow: FC<TableTaskRowProps> = ({ task }) => {
             <Trans i18nKey="Notify" />
           </button>
         )}
-        <button
-          className={classes(css['button-small-filled'])}
-          type="button"
-          disabled={!taskAwaitsRatings(task, userInfo)}
-          onClick={openModal(ModalTypes.RATE_TASK_MODAL)}
-        >
-          <Trans i18nKey="Rate" />
-        </button>
+        {userInfo && representsCustomers(userInfo, task.roadmapId) && (
```

Comment:
This disables rating for developers
```suggestion
        {userInfo &&
          (type === RoleType.Developer ||
            representsCustomers(userInfo, task.roadmapId)) && (
```

Reply:
Good catch 👍  I'll fix that